include/flatbuffers/flatbuffers.h

```diff
@@ -1557,6 +1557,7 @@ class Table {
 // C++ data structures).
 // Contains no functionality, purely documentative.
 struct NativeTable {
+  virtual ~NativeTable() {}
```

Comment:
I agree it is nicer.. but we already have some users of this code that really prefer to keep things non-virtual.